package.json

```diff
@@ -150,6 +150,7 @@
     "sass-loader": "13.1.0",
     "standard-version": "9.5.0",
     "storybook-addon-apollo-client": "4.0.13",
+    "storybook-addon-mock": "^3.2.0",
```

Comment:
Do we need to lock this version in?

Reply:
Good catch, we should since we lock everything down. I'll fix it.